DependencyInjection/Configuration.php

```diff
@@ -38,6 +38,10 @@ private function createAuthorizationServerNode(): NodeDefinition
                     ->isRequired()
                     ->cannotBeEmpty()
                 ->end()
+                ->scalarNode('pass_phrase')
```

Comment:
@alexsegura I think that `private_key_passphrase` would be more descriptive. Everything else looks good! 👍

Reply:
Done 🙂